components/inputs/inputs.css

```diff
@@ -0,0 +1,45 @@
+.input-group {
+    color: #333;
+    font-size: 0.8rem;
+    line-height: 1.2rem;
+    margin: 0 1.2rem 0.8rem;
+    width: 20%;
+}
+  
+label {
+    display: block;
+    margin-bottom: 2px;
+}
+
+input[type=text] {
```

Comment:
doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?

Reply:
> doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?

using `text` property should account for both alphanumeric and symbols, hence it covers all possible text input by user

Reply:
Yes it was on purpose written like that.

Reply:
> > doesn't this restrict the usage of input to only type text for the user? what if, they want the input of some other type, will the styles still get applied?
> 
> using `text` property should account for both alphanumeric and symbols, hence it covers all possible text input by user

yes it does cover alphanumeric values and numbers but the styles won't apply if the user(user of this component library) sets the type other than "text"(for eg: number, email, etc.)

Reply:
Got your point prachi, thanks for bringing that up.